templates/catch-all-server.conf

```diff
@@ -0,0 +1,11 @@
+# This is the default catch all server, that intercept all requests
+# not served by others server block declarations.
+server {
+    listen       80 default_server;
+    # The _ it is just one of a myriad of invalid domain
+    # names which never intersect with any real name.
+    server_name  _;
+    # 444 error code: no response, instruct the server to return no
+    # information to the client and close the connection immediately.
+    return       444;
+}
```

Comment:
I have a doubt here, if the files are loaded in an alphabetical order and I add a file called 000-redirect.conf from a custom Docker image, it should behave then like the catch-all server.